### PR TITLE
clarify interested parties

### LINKED ISSUE #87: What does it mean to be an "interested party"?
From 2021-03-15 meeting:
What does it mean to be an interested party? - create issue.  (Per above, does Frederick need to be an interested party to have voting rights?)  Does being an interested party have any obligations, since that’s what legal depts will worry about?

For companies or individuals that want to add themselves as an interested parties, does it have any legal ramifications. Currently it gives them the right to vote in elections, but nothing else is defined. 

Can we borrow something from SIG Contributor Strategy? https://github.com/cncf/sig-contributor-strategy/tree/master/governance/docs

2020-04-27:
- 404 on doc link
- new link https://github.com/cncf/sig-contributor-strategy/tree/master/governance/

### PR DESCRIPTION
**Pull request by xmulligan**
Fixes #87 (What does it mean to be an "interested party"?)

### CODE REVIEW

File: interested-parties.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Interested Parties
+The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```
Comment (xmulligan) [inline interested-parties.md]:
```suggestion

The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```

**Review by iawells — APPROVED** [review]
Will conflict with @taylor's work when merging but should work together with it; we want both.

**Review by taylor — COMMENTED** [review]
I don't think any clarification or specificity is necessary.  

The list can be used for multiple purposes, including simply showing anyone who cares that there are interested parties.

FYI, in the K8s+CNCF interested parties lists that I have seen so far, I do not see a reason for the list.

**Review by jeffsaelens — APPROVED** [review]
LGTM

**Review by Smitholi67 — COMMENTED** [review]
LGTM.  I wonder if, however, we should be listing company/organization first since that is what we are sorting on.  Just a thought.

**Comment by taylor** [comment]
I think a more general message and intent should be used.  Notice in the comment below that an interested party for the CNCF Working Group on Chaos Engineering is anyone who has ideas or concerns in Chaos Engineering.

> A WG is good mechanism, for interested parties, to share their thoughts and ideas around a concern such as Chaos Engingeering which, while a relatively new discipline, is rapidly becoming a mainstream concern for many companies adopting cloud native practices and tools.

from https://medium.com/chaosiq/federating-chaos-engineering-with-a-cncf-working-group-c08c85c2ad18

If you are interested in the topics discussed in the CNF WG you are an interested party.  The list can be used for voting of course. That can be handled in the Governance.md.

File: interested-parties.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Interested Parties
 
+The interested parties listed below are used for voting purposes as outlined in the [Governance doc](GOVERNANCE.md). See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```
Comment (taylor) [inline interested-parties.md]:
```suggestion
To express your interest in the cloud native networking topics under discussion in this working group, please add your name and organization name to this document via a Pull Request.

Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community. 

Note: adding yourself to this list does not imply any obligation (including legal) for you nor your organization.
```

**Review by taylor — CHANGES_REQUESTED** [review]

File: interested-parties.md (hunk 2 above)
Comment (taylor) [reply interested-parties.md]: @xmulligan this change would address #87 items including the concerns (via the note) while keeping it open for other uses.

Comment (iawells) [reply interested-parties.md]: I'm good with this suggestion.

**Review by taylor — APPROVED** [review]
lgtm

**Review by lixuna — APPROVED** [review]
LGTM

File: interested-parties.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Interested Parties
 
+To express your interest in the cloud native networking topics under discussion in this working group, please add your name and organization name to this document via a Pull Request.
+
+Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```
Comment (lixuna) [inline interested-parties.md]:
```suggestion
Being listed in this document may grant you some rights within the group, such as voting capabilities as outlined in the [Governance doc](GOVERNANCE.md).  See the [contributing doc](CONTRIBUTING.md) for more information about getting involved and participating in the community.
```

Comment (lixuna) [reply interested-parties.md]: remove trailing space per linter error

**Review by rabi-abdel — APPROVED** [review]
Approved

**Review by rannyh — APPROVED** [review]
LGTM

**Review by sishbi — APPROVED** [review]
LGTM

**Review by iawells — APPROVED** [review]

**Pull request merged**